rules/windows/persistence_startup_folder_scripts.toml

```diff
@@ -101,38 +105,44 @@ This rule looks for shortcuts created by wscript.exe or cscript.exe, or js/vbs s
 """
 risk_score = 47
 rule_id = "f7c4dc5a-a58d-491d-9f14-9b66507121c0"
-setup = """## Setup
-
+setup = """
 If enabling an EQL rule on a non-elastic-agent index (such as beats) for versions <8.2,
 events will not define `event.ingested` and default fallback for EQL rules was not added until version 8.2.
 Hence for this rule to work effectively, users will need to add a custom ingest pipeline to populate
 `event.ingested` to @timestamp.
+
 For more details on adding a custom ingest pipeline refer - https://www.elastic.co/guide/en/fleet/current/data-streams-pipeline-tutorial.html
 """
 severity = "medium"
+tags = [
+    "Domain: Endpoint",
+    "OS: Windows",
+    "Use Case: Threat Detection",
+    "Tactic: Persistence",
+    "Resources: Investigation Guide",
+    "Data Source: Elastic Endgame",
+    "Data Source: Elastic Defend",
+]
 tags = ["Domain: Endpoint", "OS: Windows", "Use Case: Threat Detection", "Tactic: Persistence", "Resources: Investigation Guide", "Data Source: Elastic Endgame", "Data Source: Elastic Defend", "Data Source: Sysmon"]
+
 timestamp_override = "event.ingested"
 type = "eql"
 
 query = '''
 file where host.os.type == "windows" and event.type != "deletion" and
 
-  file.extension : ("lnk", "vbs", "vbe", "wsh", "wsf", "js") and
```

Comment:
I think this actually makes it more expensive, as it will run a wildcard in all file events, instead of just in the extensions listed

Reply:
True, based on sequence of term searches it would attempt wildcard filters on any windows events. We can instead have the startup wildcard filter be applied last after the inclusive filters are done.